code/modules/surgery/advanced/bioware/cortex_folding.dm

```diff
@@ -27,11 +27,13 @@
 	display_results(user, target, "<span class='notice'>You start folding [target]'s outer cerebral cortex into a fractal pattern.</span>",
 		"<span class='notice'>[user] starts folding [target]'s outer cerebral cortex into a fractal pattern.</span>",
 		"<span class='notice'>[user] begins to perform surgery on [target]'s brain.</span>")
+	display_pain(target, "<span class='userdanger'>Your head throbs with gruesome pain, it's nearly too much to handle!</span>") //SKYRAT EDIT ADD - SURGERY PAIN
```

Comment:
Move that to modular, copy the proc over, put that line in and call ..()
```suggestion
datum/surgery_step/fold_cortex/preop(mob/user, mob/living/carbon/target, target_zone, obj/item/tool, datum/surgery/surgery)
	display_pain(target, "<span class='userdanger'>Your head throbs with gruesome pain, it's nearly too much to handle!</span>")
	..()
```
...don't accept this as a valid change, I just can't write in a different way on phone.